fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68359,6 +68359,22 @@ relationship: capable_of GO:0097206 ! nephrocyte filtration
 relationship: part_of FBbt:00005056 ! excretory system
 relationship: part_of FBbt:00005057 ! circulatory system
 
+[Term]
+id: FBbt:00047145
+name: dorsal organ cool cell
+def: "Ir21a, Ir25a and Ir93a -expressing neuron of the dorsal organ ganglion (DOG), activated by temperature decreases. All three of these Ir genes are necessary for cold-induced activity of the cell. Three of these cells are found in each DOG and they are required for cold avoidance behavior. The sensory dendrites of the three thermosensory neurons exhibit a large membrane-rich bulb before reaching the central dome, unlike the olfactory neurons in the DOG." [FlyBase:FBrf0227329, FlyBase:FBrf0232228, FlyBase:FBrf0233685]
+synonym: "DOCC" EXACT []
+relationship: capable_of GO:0050960 ! detection of temperature stimulus involved in thermoception
+relationship: expresses http://flybase.org/reports/FBgn0031209
+relationship: has_soma_location FBbt:00007227 ! dorsal organ ganglion
+relationship: overlaps FBbt:00002655 ! dorsal organ
+
+[Term]
+id: FBbt:00047184
+name: dorsal organ sensory neuron
+intersection_of: FBbt:00005124 ! sensory neuron
+intersection_of: overlaps FBbt:00002655 ! dorsal organ
+
```

Comment:
IIRC You should need to add def "."  in order for FBbt:00047184 to get an auto-def.  I think this was part of the checks in the Perl scripts we've yet to migrate to Travis.   

BTW - Wondering about lack of created_by and creation_date tags for new terms.  We probably forgot to set this up in Protege on your new machine (http://go-ontology.readthedocs.io/en/latest/Configs.html#configuring-new-entities-metadata).  I'll add this to the doc.

One other thing I totally forgot to mention.  We usually add initials to the refs for terms we've edited.  e.g. my edits have FBC:DOS.    We can use these to trace who edited what - and also for micro-attribution by linking to ORCID.  Don't worry too much if you forget though.  I haven't been very consistent about this myself.